src/kirby/components/grid/breakpoint-helper.service.tns.ts

```diff
@@ -18,10 +18,18 @@ export class BreakpointHelperService implements OnDestroy {
   private init() {
     this.currentScreenWidth = screen.mainScreen.widthDIPs;
     app.on(app.orientationChangedEvent, (args: OrientationChangedEventData) => {
-      if (this.currentScreenWidth === screen.mainScreen.widthDIPs) {
-        this.currentScreenWidth = screen.mainScreen.heightDIPs;
-      } else {
-        this.currentScreenWidth = screen.mainScreen.widthDIPs;
+      if (args.newValue === 'portrait') {
```

Comment:
burde 'portrait' ikke være en enum/const et sted i common. tænker at nogle måske kan skrive lidt forskelling 'portrait' og ''Portrait

Reply:
Faktisk så for du hjælp af VS Code her fordi den ved hvad den værdi kan være, og derfor beholdte jeg det sådan.